src/gleam/list.gleam

```diff
@@ -488,6 +488,15 @@ if javascript {
   }
 }
 
+/// Prefixes an item to a list. This can also be done using the dedicated
+/// syntax instead
+///
+///    let new_list = [1, ..existing_list]
```

Comment:
Can we use the backtick syntax for code blocks in markdown? We've had quite a few issues with the indentation amount being wrong when using this style

Reply:
@lpil For sure. Would you like me to update all of the examples? Looks like they're all using indentation at the moment.

Reply:
If you want to, but I don't mind either way :)